src/groupeddataframe/splitapplycombine.jl

```diff
@@ -1130,57 +1141,25 @@ julia> by(df, :a) do d # do syntax for the slower variant
 │ 2   │ 2     │ 8     │
 │ 3   │ 3     │ 10    │
 │ 4   │ 4     │ 12    │
-
-julia> by(df, :a, :c => x -> 2 .* x)
```

Comment:
Maybe use the same examples as for `combine`?

Reply:
OK